src/main/kotlin/lotto/ProfitRate.kt

```diff
@@ -0,0 +1,18 @@
+package lotto
```

Comment:
비즈니스 문제를 해결하는 클래스들은 `domain` 패키지에 위치하면 어떨까요?

```suggestion
package lotto.domain
```

Reply:
반영하였어요. e1fb3e2 
클래스 이름도 수정이 필요하여 함께 수정하였습니다. (`ProfitRate` -> `Profit`)